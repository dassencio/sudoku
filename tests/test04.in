2 3
1 _ _ _ _ 2
_ 3 _ 4 1 _
_ _ _ _ 3 _
_ 1 _ _ _ _
_ 5 1 _ 2 _
6 _ _ _ _ 3
