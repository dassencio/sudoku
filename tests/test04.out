1 4 5 3 6 2
2 3 6 4 1 5
4 6 2 5 3 1
5 1 3 2 4 6
3 5 1 6 2 4
6 2 4 1 5 3
